src/Generators/VsixManifestGenerator.cs

```diff
@@ -36,13 +36,15 @@ private string GenerateClass(VsixManifest manifest)
             sb.AppendLine("{");
             sb.AppendLine($"    public sealed partial class {FileNamespace}Manifest");
             sb.AppendLine($"    {{");
-            sb.AppendLine($"        public const string Id = \"{EscapeString(manifest.ID)}\";");
-            sb.AppendLine($"        public const string Name = \"{EscapeString(manifest.Name)}\";");
-            sb.AppendLine($"        public const string Description = @\"{EscapeVerbatimString(manifest.Description)}\";");
-            sb.AppendLine($"        public const string Language = \"{EscapeString(manifest.Language)}\";");
-            sb.AppendLine($"        public const string Version = \"{EscapeString(manifest.Version)}\";");
-            sb.AppendLine($"        public const string Author = \"{EscapeString(manifest.Author)}\";");
-            sb.AppendLine($"        public const string Tags = \"{EscapeString(manifest.Tags)}\";");
+            sb.AppendLine($"        public static {FileNamespace}Manifest Instance {{ get; }} = new {FileNamespace}Manifest();");
+            sb.AppendLine();
+            sb.AppendLine($"        public string Id {{ get; }} = \"{EscapeString(manifest.ID)}\";");
+            sb.AppendLine($"        public string Name {{ get; }} = \"{EscapeString(manifest.Name)}\";");
```

Comment:
The reason these are constants is such that they can be used in attributes. Why change it?

Reply:
They can be left as constants after all, my bad. I was trying to make the generator produce an interface, but it's not necessary to.  I'll undo the changes here.

Reply:
I've made two new commits.